terraform/projects/app-ecs-services/config/alerts/registers.yml

```diff
@@ -0,0 +1,9 @@
+groups:
+- name: Registers
+  rules:
+  - alert: RequestsExcess5xx
+    expr: sum(increase(requests{job="openregister-metric-exporter",status_range="5xx"}[5m])) by (app) >= 3
```

Comment:
I agree with the documentation - but I don't think it says we should change anything here.  Alerting rules and recording rules are different things: recording rules are like materialised views which should generate consistently-defined time series - so a rate time series should be rate-per-second, rather than a mix of series with rates per second, per minute, per 5 minutes, or whatever.

Alerting rules should be human-readable - in this case, it's saying "alert if the increase in errors is greater than 3 within a 5 minute window" which is far more understandable than "alert if the rate of errors per second is greater than 0.01 over a 5 minute window".  Alerting rules don't output new time series into the tsdb, so they don't have to adhere to the same consistency contraints as recording rules.